bitmap-raziyehLogan/lib/bitmap-file-transform.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const Color = require('../model/color.js');
+module.exports = exports = {};
+
+
+exports.dataArrayMap = function(colorArray){
+  var currentColor, currentColorArray = [], newColor;
+  for(var x = 0; x < colorArray.length; x += 4) {
+    currentColor = colorArray.slice(x, x + 4);
+    newColor = new Color(currentColor);
+    newColor.grayScale();
+    currentColorArray.push(newColor);
+    console.log(currentColorArray);
```

Comment:
@loganlsr @Raziyehbazargan - be sure to remove all occurrences of `console.log()` from future projects - these are totally fine to keep in your code, as long as they exist on a separate branch, possibly even used just for testing  ie: `features/transform-testing`